otx/algorithms/classification/adapters/mmcls/data/datasets.py

```diff
@@ -281,6 +281,7 @@ def evaluate(
                 if k in metrics:
                     eval_results[k] = v
 
+        eval_results["accuracy"] = eval_results["accuracy-mlc"]
```

Comment:
Hi @eunwoosh. Earlier in this file we have a [line ](https://github.com/openvinotoolkit/training_extensions/pull/1646/files#diff-24b4265f1eaea95b802d58f85dc3dd3d9b233776a1dc53b19d842865ba927593L247)
```
if "accuracy-mlc" in metrics:
```
That means `accuracy-mlc` conditionally included into `eval_results` and sometimes this line could cause a failure.
I'll fix that in [#1638](https://github.com/openvinotoolkit/training_extensions/pull/1638)

Reply:
I forget about that. Thanks for pointing it.